target/linux/ramips/dts/mt7628an_tuoshi_lt22m.dts

```diff
@@ -0,0 +1,186 @@
+#include "mt7628an.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	compatible = "tuoshi,lt22m", "mediatek,mt7628an-soc";
+	model = "Tuoshi LT22M 4G Wireless Router";
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "blue:wifi";
+			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+		};
+
+		lan0 {
+			label = "blue:lan0";
+			gpios = <&gpio 43 GPIO_ACTIVE_LOW>;
+		};
+
+		lan1 {
+			label = "blue:lan1";
+			gpios = <&gpio 42 GPIO_ACTIVE_LOW>;
+		};
+
+		2g {
+			label = "blue:2g";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+		};
+
+		3g {
+			label = "blue:3g";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+		};
+
+		4g {
+			label = "blue:4g";
+			gpios = <&gpio 37 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		modem_enable {
+			gpio-export,name = "modem_enable";
+			gpio-export,output = <0>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
It makes sense to add a corresponding `ucidef_add_gpio_switch` entry in `/etc/board.d/03_gpio_switches`.

Reply:
I do not exactly understand what this means and what's the purpose.

Reply:
Please see here:
https://github.com/openwrt/openwrt/pull/14018/commits/4f72d07e7b525b3184b26144506530ba86cbe869#diff-bf2e57f8034bb367b235407b1b0eb7dafe19ef0e188f7f49aeaf55cd71b32f47